### PR TITLE
feat(core): Add MLS team feature config (fs-653)

### PR DESCRIPTION
**Pull request by thisisamir98**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/FS-653" title="FS-653" target="_blank">TaskFS-653</a>  [Web] parse the MLS feature config flag
  </td></table>
  <br />
 

#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [X] contains a reference JIRA issue number like `SQPIT-764`
  - [X] answers the question: _If merged, this PR will: ..._ ³

### CODE REVIEW

File: src/script/team/TeamService.ts [hunk 1]
```diff
@@ -107,6 +107,12 @@ export class TeamService {
         [FEATURE_KEY.SSO]: {
           status: FeatureStatus.ENABLED,
         },
+        [FEATURE_KEY.MLS]: {
+          config: {
+            allowList: [],
+          },
+          status: FeatureStatus.ENABLED,
```
Comment (atomrc) [inline src/script/team/TeamService.ts]: Let's have it disabled by default

File: src/script/team/TeamState.ts [hunk 2]
```diff
@@ -113,6 +114,7 @@ export class TeamState {
       // TODO connect to video calling feature config
       () => true || this.teamFeatures()?.videoCalling?.status === FeatureStatus.ENABLED,
     );
+    this.isMLSEnabled = ko.pureComputed(() => true || this.teamFeatures()?.mls?.status === FeatureStatus.ENABLED);
```
Comment (atomrc) [inline src/script/team/TeamState.ts]: Seems like there is some debugging code here ;)

**Review by atomrc — CHANGES_REQUESTED** [review]

File: src/script/team/TeamState.ts (hunk 2 above)
Comment (thisisamir98) [reply src/script/team/TeamState.ts]: https://github.com/wireapp/wire-webapp/pull/13131/commits/5a52d9d38f95740f49c57e26247d43940e3ddacf

File: src/script/team/TeamService.ts (hunk 1 above)
Comment (thisisamir98) [reply src/script/team/TeamService.ts]: https://github.com/wireapp/wire-webapp/pull/13131/commits/64aaf1b2e7c7e2263a2464962be84e3688d63a7b

**Review by atomrc — APPROVED** [review]

**Pull request merged**